easybuild/easyblocks/generic/intelbase.py

```diff
@@ -202,36 +204,40 @@ def setup_local_home_subdir(self):
         except OSError, err:
             raise EasyBuildError("Failed to symlink %s to %s: %s", self.home_subdir_local, self.home_subdir, err)
 
-    def prepare_step(self):
+    def prepare_step(self, requires_runtime_license=True):
         """Custom prepare step for IntelBase. Set up the license"""
         super(IntelBase, self).prepare_step()
 
-        default_lic_env_var = 'INTEL_LICENSE_FILE'
-        lic_specs, self.license_env_var = find_flexlm_license(custom_env_vars=[default_lic_env_var],
-                                                              lic_specs=[self.cfg['license_file']])
+        # Decide if we need a license or not (default is True because of defaults of individual Booleans)
+        self.requires_runtime_license = self.cfg['requires_runtime_license'] and requires_runtime_license
```

Comment:
please initialise `self.requires_runtime_license` in the constructor `__init__`, probably to `True`; this is import for `--module-only` for example